fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarMenuController.kt

```diff
@@ -483,6 +489,12 @@ class DefaultBrowserToolbarMenuController(
                 Events.browserMenuAction.record(Events.BrowserMenuActionExtra("set_default_browser"))
             is ToolbarMenu.Item.RemoveFromTopSites ->
                 Events.browserMenuAction.record(Events.BrowserMenuActionExtra("remove_from_top_sites"))
+
+            ToolbarMenu.Item.Translations -> Events.browserMenuAction.record(
```

Comment:
I think this might require a data review request and an entry in [metrics.yaml](https://searchfox.org/mozilla-mobile/rev/b4fff7f0b3d73d23a8baf7711dc8e0163518d2a1/firefox-android/fenix/app/metrics.yaml#136-143).

@rocketsroger does adding a new `extra_keys` option to the browser menu require a new data review?

Reply:
No, but please add this to the [extra keys description](https://searchfox.org/mozilla-mobile/rev/b4fff7f0b3d73d23a8baf7711dc8e0163518d2a1/firefox-android/fenix/app/metrics.yaml#135) so `translations` is stated there as well.

Reply:
Done